native/packages/react-native-bpk-component-theming/readme.md

```diff
@@ -0,0 +1,45 @@
+# react-native-bpk-component-theming
+
+> Backpack React Native theming component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-theming --save-dev
+```
+
+## Usage
+
+```js
+import { Component } from 'react';
+import { View } from 'react-native';
+import BpkThemeProvider from 'react-native-bpk-component-theming';
+
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+const ThemeableComponent = withTheme((props) => {
```

Comment:
Is this really how we wann document this? Isn't it better to document it with a Backpack component that's themable. e.g

```
      <BpkThemeProvider theme={themes}>
        <BpkButton>Book</BpkButton>
      </BpkThemeProvider>
```

Reply:
@matthewdavidson and I spoke about this - the plan is to update the docs as soon as we have a themeable component to use it.

Reply:
I guess we could put a button in there ahead of time, even though it isn't strictly themeable yet. It will be soon...?